docs/readmes/README.featureflags.md

```diff
@@ -0,0 +1,23 @@
+# Feature flags
+> Available from 1.5.3 release
```

Comment:
It's possible we add feature flags in the future releases as well. I think we should move this under each feature flag section.